ember/app/ui/cases/index/controller.js

```diff
@@ -71,7 +77,7 @@ export default class CasesIndexController extends Controller {
       filters.push({
         searchAnswers: [
           {
-            forms: (await this.filteredForms.fetch()).map((form) => form.slug),
```

Comment:
Why did you put this in a separate `trackedFunction`?

Reply:
Moved into `filteredForms` service

Reply:
@velrest Can you please review again /approve? Wanna merge this today